config.json

```diff
@@ -0,0 +1,10 @@
+{
+        "auths": {
+                "quay.io": {
+                        "auth": "b3BlbnNoaWZ0LXJlbGVhc2UtZGV2K3lzaG5haWRtMXkxdTZhempjaGJ0cm5hZDFkb3U1Z3FtcnliOlVBNjU0SVZFQzZPMFFBM0ZJTDA2WjY1WDRSSjM0WElNM0JYRVkzVFBCVU9IUkZESEhRSkkxMFFJNUI5MEY1Tks="
```

Comment:
Why do you need it and why not take it from the install config?

Reply:
i needed it to extract openshift-baremetal-installer for 4.6. I will remove it, and when we move to 4.6 will use pull secret from install-config